src/transformers/image_utils.py

```diff
@@ -243,7 +243,7 @@ def make_flat_list_of_images(
     if (
         isinstance(images, (list, tuple))
         and all(isinstance(images_i, (list, tuple)) for images_i in images)
-        and all(is_valid_list_of_images(images_i) for images_i in images)
+        and any(is_valid_list_of_images(images_i) for images_i in images)
```

Comment:
hmm, I would better extend it to support an empty list, otherwise, we might have something not relevant in any of the lists, e.g.

```suggestion
        and all(is_valid_list_of_images(images_i) or not images_i  for images_i in images)
```